Add selectively boot apps page to 1.1 guides

davydovanton opened the pull request:
## 1.1 version
![screenshot 2018-02-06 17 07 51](https://user-images.githubusercontent.com/1147484/35863638-58970376-0b60-11e8-9370-1267faa6cae0.jpeg)

## 1.0 version
![screenshot 2018-02-06 17 07 57](https://user-images.githubusercontent.com/1147484/35863651-5e1a0dac-0b60-11e8-8a7c-cd9aa8746038.jpeg)

davydovanton committed:
Add selectively boot apps page to 1.1 guides

AlfonsoUceda reviewed (approved):
Great @davydovanton 👍

cllns committed:
Re-work text for 'Selectively Boot Apps'

Fixed a couple grammar things, but also changed the examples to improve
clarity... IMO :)

cllns commented:
Thanks for writing this @davydovanton. I just re-worked some of the text and committed it. I fixed some grammar, but also changed some examples to provide more clarity (like avoiding adding A, B, C servers). What do you think about my changes?

davydovanton commented:
@cllns awesome work, really like it 💯 
Let's merge it! :shipit:
